pay-api/src/pay_api/models/eft_refund_email_list.py

```diff
@@ -0,0 +1,55 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Model to handle all operations related to Fee related to accounts."""
+from __future__ import annotations
+
+from sql_versioning import Versioned
+
+from .base_model import BaseModel
+from .db import db
+
+
+class EFTRefundEmailList(Versioned, BaseModel):
+    """This class manages all of the Expense Authority emails."""
+
+    __tablename__ = 'eft_refund_email_list'
+    # this mapper is used so that new and old versions of the service can be run simultaneously,
+    # making rolling upgrades easier
+    # This is used by SQLAlchemy to explicitly define which fields we're interested
+    # so it doesn't freak out and say it can't map the structure if other fields are present.
+    # This could occur from a failed deploy or during an upgrade.
+    # The other option is to tell SQLAlchemy to ignore differences, but that is ambiguous
+    # and can interfere with Alembic upgrades.
+    #
+    # NOTE: please keep mapper names in alpha-order, easier to track that way
+    #       Exception, id is always first, _fields first
+    __mapper_args__ = {
+        'include_properties': [
+            'id',
+            'first_name',
+            'last_name',
+            'email'
+        ]
+    }
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
```

Comment:
What are the relationships for this table? Seems fairly barebones now

Reply:
no it's standalone now, I only have list of names from the ticket